pkg/options/options.go

```diff
@@ -79,6 +80,7 @@ func (o *Options) AddFlags() {
 
 	o.flags.StringVar(&o.Apiserver, "apiserver", "", `The URL of the apiserver to use as a master`)
 	o.flags.StringVar(&o.Kubeconfig, "kubeconfig", "", "Absolute path to the kubeconfig file")
+	o.flags.StringVar(&o.TLSConfig, "tls-config", "", "Path to TLS config file")
```

Comment:
Can we also maintain lexical order of the flags now that we are making this change?
```suggestion
	o.flags.StringVar(&o.TLSConfig, "tls-config", "", "Path to the TLS config file")
```